EventListener/ExceptionLogSubscriber.php

```diff
@@ -46,6 +47,8 @@ public function onException($event): void
                 'stack' => $exception->getTraceAsString(),
             ]
         );
+        // TODO: not sure if the tag needs to be added here or in the FinishXXXSpan listener (to avoid duplicate)
+        $this->tracing->setTagOfActiveSpan(ERROR, true);
```

Comment:
Should add a test for this

Reply:
I removed the error tag handling in case of an exception, it should be handled when finishing span.
Handling it here would introduce a duplicate tag, and it would be wrong in case the exception is recovered.